linux_os/guide/services/ntp/chronyd_client_only/rule.yml

```diff
@@ -20,6 +20,7 @@ identifiers:
 
 references:
     ospp: FMT_SMF_EXT.1
+    srg: SRG-OS-000096-GPOS-00050
```

Comment:
Tentative NACK. Does DoD truly prohibit running time servers?

Reply:
No, but this rule prevents Chrony from acting like a server when it should be a client only.

Reply:
The NTP protocol is very complicated and subject to programming mistakes. (Look at the CVEs around it.) By default the daemon will be a time server unless steps are taken to harden it. So, this falls under the principle of least surprise. I didn't know I had an accessible open port that serves time. Also, there are attacks that are prevented or made more difficult by not having access to precise time on a system. And then there are the amplification attacks caused by people not knowing this port is open and will respond to any target.